<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GlobalPropertiesWidget</class>
 <widget class="QWidget" name="GlobalPropertiesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_7" >
       <property name="minimumSize" >
        <size>
         <width>500</width>
         <height>16</height>
        </size>
       </property>
       <property name="title" >
        <string>General settings</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="waveFormMergedCheckBox" >
          <property name="text" >
           <string>Draw stereo tracks as one AudioWave</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QCheckBox" name="loadLastProjectCheckBox" >
          <property name="text" >
           <string>Load last used project at startup</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="waveFormRectifiedCheckBox" >
          <property name="text" >
           <string>Draw AudioWaves Rectified</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6" >
       <property name="title" >
        <string>New song options</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Bitstream Vera Sans; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Number of tracks&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numberOfTrackSpinBox" >
          <property name="maximum" >
           <number>1024</number>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4_2" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Bitstream Vera Sans; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Horizontal zoom level&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="defaultHZoomLevelComboBox" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <item>
           <property name="text" >
            <string>1:8</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:20</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:64</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:128</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:256</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:512</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:1024</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:2048</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:4096</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:8192</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1:16384</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>224</height>
        </size>
       </property>
       <property name="title" >
        <string>Hardware</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item rowspan="2" row="0" column="0" >
             <widget class="QLabel" name="audiocardLabel" >
              <property name="maximumSize" >
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap/>
              </property>
             </widget>
            </item>
            <item rowspan="2" row="1" column="2" >
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_8" >
                <property name="text" >
                 <string>Buffer latency</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="latencyLabel" >
                <property name="text" >
                 <string>latency (ms)</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1" >
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2_2" >
                <property name="text" >
                 <string>Sample rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="defaultSampleRateComboBox" >
                <item>
                 <property name="text" >
                  <string>22050</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>32000</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>44100</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>48000</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>88200</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>96000</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item rowspan="2" row="1" column="1" >
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6" >
                <property name="text" >
                 <string>Audio Driver</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="audioDriverBackendComboBox" />
              </item>
             </layout>
            </item>
            <item row="0" column="2" >
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_7" >
                <property name="text" >
                 <string>Buffer size</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="bufferSizeComboBox" >
                <item>
                 <property name="text" >
                  <string>16</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>32</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>64</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>128</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>256</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>512</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>1024</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>2048</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>4096</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="2" >
             <widget class="QPushButton" name="deviceApplyButton" >
              <property name="text" >
               <string>Restart Driver</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="cardNameLabel" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Name</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="keyboardLabel" >
              <property name="maximumSize" >
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Bitstream Vera Sans; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Double fact timeout (ms)&lt;/p>&lt;/body>&lt;/html></string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="doubleFactTimeoutSpinBox" >
                <property name="maximum" >
                 <number>300</number>
                </property>
                <property name="minimum" >
                 <number>130</number>
                </property>
                <property name="value" >
                 <number>200</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Bitstream Vera Sans; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Hold timeout (ms)&lt;/p>&lt;/body>&lt;/html></string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="holdTimeoutSpinBox" >
                <property name="maximum" >
                 <number>300</number>
                </property>
                <property name="minimum" >
                 <number>130</number>
                </property>
                <property name="value" >
                 <number>200</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="defaultsButton" >
         <property name="text" >
          <string>Defaults</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton" >
         <property name="text" >
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="discardButton" >
         <property name="text" >
          <string>Discard</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
